content/news/2020-08-10-introducing-bevy/index.md

```diff
@@ -895,7 +920,7 @@ This will load new versions of assets whenever their files have changed.
 
 #### Adding New Asset Types
 
-To add a new asset type, implement the {{rust_type(type="trait" crate="bevy_asset" version="0.1.0" name="AssetLoader" no_mod=true)}} trait. This tells Bevy what file formats to look for and how to translate the file bytes into the given asset type.
+To add a new asset type, implement the `AssetLoader` trait. This tells Bevy what file formats to look for and how to translate the file bytes into the given asset type.
```

Comment:
This trait does not exist in the public API of Bevy 0.1. As a result, I cannot replace this link.

Reply:
https://docs.rs/bevy_asset/0.1.0/bevy_asset/trait.AssetLoader.html